hapi-utils/src/main/java/com/hedera/services/legacy/proto/utils/CommonUtils.java

```diff
@@ -64,7 +64,7 @@ public static void napMillis(long timeInMillis) throws InterruptedException {
    * Sleep given seconds.
    */
   public static void nap(int timeInSec)  throws InterruptedException {
-    Thread.sleep(timeInSec * 1000);
+    Thread.sleep((long) timeInSec * 1000);
```

Comment:
No, do a `* 1000L`! Or as I remember we have those conversion constants to be reused.

Reply:
Fixed it ! Thanks !